css/updater.css

```diff
@@ -68,7 +68,12 @@
 }
 
 .updater-update {
-	padding: 1em 0.75em;
+	padding: 10px 20px;
 	background-color: rgba(240, 240, 240, 0.9);
 	max-width: 750px;
-}
\ No newline at end of file
+}
+
+.updater-update h3 {
+	font-weight:bold;
```

Comment:
That is, actually there should be no custom h styles at all. The h3 from core should suffice.